### DISCUSSION TITLE
False triggering

### DISCUSSION THREAD

**Discussion by devdems** [ZoneMinder/zoneminder#3803] [Q&A] [answered]
Hi, how can I configure Zoneminder not to trigger the alarm when the camera has a buffer or connection issue when the whole image turns gray color.

![image](https://github.com/ZoneMinder/zoneminder/assets/8215353/3c247d5c-0327-4c39-a0b0-42577b0ebd05)

**Comment by connortechnology** [comment, marked as answer]
You can set Alarm Frame Count > 1 which may mitigate it a bit... but really you should fix the image corruption.
